src/templates/timeslider.html

```diff
@@ -82,7 +82,7 @@
           <div class="editbarright toolbar" id="editbar">
             <% e.begin_block("timesliderEditbarRight"); %>
             <ul>
-              <%- toolbar.menu(settings.toolbar.timeslider, true) %>
+              <%- toolbar.menu(settings.toolbar.timeslider, true, false) %>
```

Comment:
In timeslider.html it gets the `false` value (a boolean).
At this point it makes sense for it to be a string literal, too (in TypeScript it would be an `enum`).

Should we using one of `'left'`/`'right'` because it conveys a meaning of some sort or do we need a third one (`'whatever'`)?

Reply:
'right' would work but you might also go 'timesliderRight' , your call :)

Reply:
timesliderRight